### PR TITLE
Sort entries by key & better output messages

### PR DESCRIPTION
**Pull request by blumentali**
This PR includes:
1. sort by key and not comment
2. remove "x is internationalized but isn't one of [JTLabel, ...]" warnings
3. nicer logging
4. some cleanups

Below is an example of the output when running scripts/genstrings.sh from SimplyPiano:
```
===== Fetching strings from content and configuration =====

===== Updating asla server repo =====
REPO_NAME: asla_server, BRANCH_NAME: master

Already up to date.
Current branch master is up to date.
Fetching strings from asla server code

===== Appending manually retained strings =====

===== Appending android strings =====
REPO_NAME: SimplyPianoDroid, BRANCH_NAME: better-translations

Current branch better-translations is up to date.
generating layout strings
generating content overrides strings
generating java source code strings
jtlocalize: ------ genstrings warnings: ------
genstrings: error: bad entry in file scripts/../../SimplyPianoDroid/scripts/localization/../../ingame/src/com/joytunes/common/localization/Localize.java (line = 43): Argument is not a literal string.

Done

===== Generating localization =====
jtlocalize: ------ genstrings warnings: ------
genstrings: error: bad entry in file scripts/../PianoMoments/PianoMoments/Util/LocalizationUtils.swift (line = 12): Argument is not a literal string.
genstrings: error: bad entry in file scripts/../SimplyPiano/Util/SwiftJTLocalizeUtils.swift (line = 12): Argument is not a literal string.
genstrings: error: bad entry in file scripts/../SimplyPiano/Model/Reminders/RemindersManager.swift (line = 110): Argument is not a literal string.
genstrings: error: bad entry in file scripts/../SimplyPiano/Model/Reminders/RemindersManager.swift (line = 115): Argument is not a literal string.
genstrings: error: bad entry in file scripts/../Pods/AWSMobileAnalytics/AWSMobileAnalytics/Internal/AWSMobileAnalyticsErrorUtils.m (line = 24): Argument is not a literal string.


===== Updating server repo =====
REPO_NAME: server, BRANCH_NAME: master

error: cannot pull with rebase: You have unstaged changes.
error: please commit or stash them.
WARNING: Pull Failed, proceeding with local copy
Fetching strings from server code

===== Prepare diff =====
Done!
```

### CODE REVIEW

**Comment by blumentali** [comment]
Noaaaaaaam! You of all people should know I'm not a true friend

File: localization_flow/jtlocalize/genstrings.py [hunk 1]
```diff
@@ -99,11 +92,10 @@ def generate_strings(project_base_dir, localization_bundle_path, tmp_directory,
 
     genstrings_rc = genstrings_process.returncode
     if genstrings_rc != 0:
-        logging.fatal("genstrings returned %d, aborting run!", genstrings_rc)
+        logging.fatal("\n!!!!!!!!! genstrings returned %d, ABORTING RUN !!!!!!!!!\n", genstrings_rc)
```
Comment (inbalg) [inline localization_flow/jtlocalize/genstrings.py:95]: I wonder if you can make it red :)

File: localization_flow/jtlocalize/core/handle_duplicates_in_localization.py [hunk 2]
```diff
@@ -29,30 +29,21 @@ def handle_duplications(file_path):
...
         else:
             loc_obj = LocalizationEntry(comments, key, value)
             keys_to_objects[key] = loc_obj
-            section_file_elements.append(loc_obj)
+            file_elements.append(loc_obj)
 
-    # Adding last section
-    for elem in sorted(section_file_elements, key=lambda x: x.comments[0]):
-        file_elements.append(elem)
+    # Sort by key
```
Comment (inbalg) [inline localization_flow/jtlocalize/core/handle_duplicates_in_localization.py:45]: 🔥

Comment (nirni) [reply localization_flow/jtlocalize/core/handle_duplicates_in_localization.py:45]: 🔥🔥

File: localization_flow/jtlocalize/genstrings.py (hunk 1 above)
Comment (blumentali) [reply localization_flow/jtlocalize/genstrings.py:95]: Could be nice, but I'll leave it for next time

**Pull request merged**